examples/src/examples/PAEngineMultipleUnitExample.java

```diff
@@ -51,6 +51,9 @@ public class PAEngineMultipleUnitExample {
   private static String GROUP_NAME = "Economic Sector - FactSet";
   private static String GROUP_CATEGORY = "FactSet";
   private static String GROUP_DIRECTORY = "Factset";
+  private static String Pricing_Source_Name = "MSCI - Gross";
+  private static String Pricing_Source_Category = "MSCI";
+  private static String Pricing_Source_Directory = "Equity";
```

Comment:
Looks like the naming convention is incorrect.

Reply:
Updated naming conventions